test/addons/rook-pool/storage-class.yaml

```diff
@@ -20,3 +20,4 @@ parameters:
     csi.storage.k8s.io/node-stage-secret-namespace: rook-ceph
     csi.storage.k8s.io/fstype: ext4
 reclaimPolicy: Delete
+allowVolumeExpansion: true
```

Comment:
changing pvc size will break after failover or relocate since we don't update the pvc/pv after such changes. Do you plan to to update protected pvcs/pvs in s3 after changes?

Reply:
You are right, there will be inconsistencies after the failover/relocate, this needs to be fixed.  i am working on this issue https://github.com/RamenDR/ramen/issues/74. this is part of the `KubobjectProtection` enhancement.